src/Api/Controllers/PoliciesController.cs

```diff
@@ -47,6 +48,19 @@ public class PoliciesController : Controller
             "OrganizationServiceDataProtector");
     }
 
+    [HttpGet("/policies")]
+    public async Task<ListResponseModel<PolicyResponseModel>> Get()
+    {
+        var user = await _userService.GetUserByPrincipalAsync(User);
+        if (user == null)
+        {
+            throw new BadRequestException("User not found.");
+        }
+        var policies = await _policyRepository.GetManyByUserIdAsync(user.Id);
+        var responses = policies.Select(p => new PolicyResponseModel(p));
+        return new ListResponseModel<PolicyResponseModel>(responses);
+    }
```

Comment:
I don't understand why this endpoint is needed, although I haven't reviewed the clients PR yet. So this is more of a reminder for myself to come back and look at this again later.

If we do need it, then I'm not sure why it's getting all policies and not just returning the single MP requirements object like the other responses are.

Reply:
This was necessary during the original implementation of this PR (fetching the policies in a separate HTTP request). After moving the policies into the Identity responses, I found that it was still required by a certain edge case in the CLI. 

However, now that you bring it up, I might be able to avoid it entirely in the CLI as well, and will look into removing this entirely.

Reply:
I was able to tweak the CLI implementation to avoid needing this endpoint and I've gone ahead and removed it from the PR.

Reply:
Great! Even better 😁